src/pages/[guild]/collect/[chain]/[address].tsx

```diff
@@ -236,22 +218,75 @@ const getStaticProps: GetStaticProps<Props> = async ({ params }) => {
       notFound: true,
     }
 
-  const endpoint = `/v2/guilds/guild-page/${urlName}`
-  const guild: Guild = await fetcher(endpoint).catch((_) => ({}))
+  const guildPageEndpoint = `/v2/guilds/guild-page/${urlName}`
 
-  if (!guild?.id)
+  let publicGuild: Guild, guild: Guild
+  try {
+    ;[publicGuild, guild] = await Promise.all([
+      fetcher(guildPageEndpoint),
+      fetcher(
```

Comment:
I'm wondering if later on we should add an api-key protected endpoint specifically for this usage, so we don't request the whole guild-page 🤔